.github/workflows/php_unit_test.yml

```
@@ -0,0 +1,70 @@
+
+permissions:
+  contents: read
+
+jobs:
+    php_unit:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                php-versions: ['7.4', '8.0', '8.2']
+                wp-versions: ['6.7', '6.8']
+        name: PHP Unit Test on PHP ${{ matrix.php-versions }} / WP ${{ matrix.wp-versions }} Test on ${{ matrix.operating-system }}
```

actionlint (1.7.12)

[error] property "operating-system" is not defined in object type {php-versions: number; wp-versions: number}

(expression)